examples/cuda/default.cu

```diff
@@ -1,6 +1,6 @@
 // Type your code here, or load an example.
-__global__ void square(int *array, int n) {
-    int tid = blockIdx.x;
+__global__ void square(int* array, int n) {
```

Comment:
Moved pointer to the left to make it more in-line with the C code examples, I see other cuda examples here put the pointer in the middle though